server/engine/src/main/java/org/eclipse/lsp/cobol/core/engine/dialects/idms/IdmsDialect.java

```diff
@@ -88,11 +86,17 @@ public List<SyntaxError> extend(DialectProcessingContext context) {
                     errors, cb,
                     context.getProgramDocumentUri(),
                     context.getExtendedSource().getUri(),
-                    context.getCopybookConfig(),
-                    copybookStack));
+                    context.getCopybookConfig()));
     return errors;
   }
 
+  private void insertIdmsCopybook(ExtendedSource extendedSource, DocumentMap currentMap, List<SyntaxError> errors,
```

Comment:
@Nurkambay this method is OK, it just call onther one with empty LinkedList as copybookStack

Reply:
👍